client/src/templates/Challenges/components/tool-panel.tsx

```diff
@@ -1,8 +1,5 @@
-import {
-  Button,
-  DropdownButton,
-  MenuItem
-} from '@freecodecamp/react-bootstrap';
+import { Button } from '@freecodecamp/react-bootstrap';
+import { Dropdown, MenuItem } from '@freecodecamp/ui';
```

Comment:
```suggestion
import { Dropdown, MenuItem } from '@freecodecamp/ui';
import { faExternalLinkAlt } from '@fortawesome/free-solid-svg-icons';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```
We'll need these if we add the icon to the Get a Hint menu item.